.github/workflows/test.yml

```
@@ -69,7 +69,7 @@ jobs:
           for dir in ./projects/*/; do
             if [ -f "$dir/Cargo.toml" ]; then
               echo "🔧 Building in $dir"
-              (cd "$dir" && cargo rustc --target wasm32-unknown-unknown -- -D warnings && cargo build --target wasm32-unknown-unknown) || exit 1
+              (cd "$dir" && cargo build --target wasm32-unknown-unknown && cargo rustc --target wasm32-unknown-unknown -- -D warnings) || exit 1
```

Can we build each example using the `--release` flag too? I think we could just add:
```suggestion
              (cd "$dir" && cargo build --target wasm32-unknown-unknown && cargo rustc --target wasm32-unknown-unknown -- -D warnings) || exit 1
              (cd "$dir" && cargo build --target wasm32-unknown-unknown --release && cargo rustc --target wasm32-unknown-unknown --release -- -D warnings) || exit 1
```